Header header

sensor_msgs/Image thermal
sensor_msgs/Image rgb_front
sensor_msgs/Image rgb_right
sensor_msgs/Image rgb_back
sensor_msgs/Image rgb_left
sensor_msgs/PointCloud2 points
